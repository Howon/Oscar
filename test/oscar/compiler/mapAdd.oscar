def main() => unit = {
  map<int, double> tMap = map<int, double>[0 -> 1.1, 4 -> 5.3, -3 -> 5.3];
  println(mapAdd(tMap, 2 -> -1.3) == map<int, double>[0 -> 1.1, 4 -> 5.3, -3 -> 5.3, 2 -> -1.3]);
  println(tMap == map<int, double>[0 -> 1.1, 4 -> 5.3, -3 -> 5.3]);
  println(mapAdd(tMap, 0 -> 1.1) == map<int, double>[0 -> 1.1, 4 -> 5.3, -3 -> 5.3]);
}