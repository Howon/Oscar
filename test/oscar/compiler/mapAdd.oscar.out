[0 -> 1.1, 4 -> 5.3, -3 -> 5.3, 2 -> -1.3]
[0 -> 1.1, 4 -> 5.3, -3 -> 5.3]